<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_GestionMem</class>
 <widget class="QDialog" name="f_GestionMem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de mémoire</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/Images/App/sim.png</normaloff>:/Images/App/sim.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Configuration">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Banque n° 3 :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="LB_SlotBank3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="CB_Bank2">
           <item>
            <property name="text">
             <string>Aucune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C128</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C256</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="CB_Bank3">
           <item>
            <property name="text">
             <string>Aucune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C128</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C256</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="LB_SlotBank2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Banque n° 1 :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mémoire interne de l'Arduino</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="LB_SlotBank1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Banque n° 0 : </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Banque n° 2 :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="CB_Bank0">
           <item>
            <property name="text">
             <string>Aucune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C128</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C256</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="CB_Bank1">
           <item>
            <property name="text">
             <string>Aucune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C128</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24C256</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>1 slot mémoire</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="LB_SlotBank0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="Btn_Format_Arduino">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Formater</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="Btn_Format_Bank0">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Formater</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="Btn_Format_Bank1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Formater</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="Btn_Format_Bank2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Formater</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="Btn_Format_Bank3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Formater</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCommandLinkButton" name="ActiverAutorun">
           <property name="text">
            <string>Activer l'exécution auto</string>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/Images/App/add.png</normaloff>:/Images/App/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="DesactiverAutorun">
           <property name="text">
            <string>Désactiver l'exécution auto</string>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/Images/App/delete.png</normaloff>:/Images/App/delete.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="LancerProg">
         <property name="text">
          <string>Lancer les programmes compilé</string>
         </property>
         <property name="icon">
          <iconset resource="../ressources.qrc">
           <normaloff>:/Images/App/play.png</normaloff>:/Images/App/play.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Visualisation">
      <attribute name="title">
       <string>Visualisation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListView" name="listView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_Actualiser">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/Images/App/actualier.png</normaloff>:/Images/App/actualier.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>f_GestionMem</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>f_GestionMem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
